src/dashboard/src/Layout/Content.tsx

```diff
@@ -34,7 +35,7 @@ const Content: FunctionComponent = ({ children }) => {
   return (
     <Box flex={1} py={3} className={styles.root}>
```

Comment:
```suggestion
    <Box flex={1} py={3} pt={0} className={styles.root}>
```

instead of the top change.